htdocs/core/class/html.formfile.class.php

```diff
@@ -1290,7 +1290,7 @@ public function list_of_documents($filearray, $object, $modulepart, $param = '',
 				if ($file['name'] != '.'
 						&& $file['name'] != '..'
 						&& !preg_match('/\.meta$/i', $file['name'])) {
-					if ($filearray[$key]['rowid'] > 0) {
+					if (!empty($filearray[$key]['rowid'])) {
```

Comment:
must use
if (!empty($filearray[$key]['rowid'] && $filearray[$key]['rowid'] > 0) {

to have same effect.